luthier/config/src/tables.lua

```diff
@@ -136,19 +136,43 @@ M.physical = {
     Logical tables are defined in table M.logical. They map logical table
     names to their configuration:
 
-    * name - The name of the table
-        Defaults to the configuration's key
+    * The table is keyed on the name of the logical table
     * description - Brief documentation about the logical table.
         Defaults to the configuration's key
-    * metrics - A set of API metrics' name for this logical table, these 
-        should be a subset of the metrics configured in metrics.lua.
+    * type - [Mandatory] The nature of this logical table, used to build the table internally.
+        Currently supports:
+            * abridged - A simple set of rules just enough to describe a logical table correctly.
+                if 'abridged' is selected, the following fields will become mandatory in the config:
+                    * metrics
+                    * dimensions
+                    * granularities
+                    * physicalTables
+                    * dateTimeZone
+                the following fields will be ignored:
+                    * category
+                    * longName
+                    * retention
+                    * description
+                    * apiFilters
+            * comprehensive - a list of rules to fully configure the logicalTable. This is not implemented yet.
+        Defaults to 'abridged'.
```

Comment:
do we allow default factories?  I didn't think we did.

Reply:
We did mention that once when we had the talk about defaulting dimension factory to KeyValueStoreDimensionFactory. 
Not sure if this really makes sense here as well (although if we do decide to make a defaulting happen, I will remove the [mandatory] tag in this doc.

Reply:
I think we should default the type in cases where it makes sense. We expect the vast majority of the time for people to use only one LogicalTable type, the one provided by Fili.

Reply:
I agree. This will be more addressed and documented when we get to the luaDefaulting PR, which I hope to start after this PR.